data/pipe_output.txt

```diff
@@ -0,0 +1,3 @@
+This is the first line
+This is the second line
+This is the last line
```

Comment:
I recommend creating output file in your programm instead of having them here, if you decide to have them here, maybe leave them empty so the user can try them.

Reply:
It shouldn't be a problem as they're cleared before the IPC is executed (see Karol's remark above).

I prefer to leave them in place as then I'm able to always have them there when needed :)